crates/nostr-sqldb/migrations/sqlite/2025-04-11-095120_events/up.sql

```diff
@@ -0,0 +1,27 @@
+-- The actual event data
+CREATE TABLE events (
+    id VARCHAR(64) PRIMARY KEY,
```

Comment:
I would store the 32-byte array here, instead of the hex. The same for the pubkey and signature (64-byte array), also in the other schemas. What do your think?

Reply:
I'll have to check whether there is support for binary indexes in all dbs and if there are any performance penalties. First look seem to be doable. Debugging the data (with other sql clients) will get a bit harder though.

Reply:
Regarding the debugging in SQL clients: I know some clients display the bytes as hex, but not sure if all. I've tried DBeaver and JetBrains Database tool, and both display the event ID and public key BLOBs as hex.

Reply:
Yes at least PG is storing small binaries as hex values anyhow. I have dropped the signature column and set the column types to be binary. Testing with PG seems to have no issues for the other backends I'll have to check when I add the impls.